core/templates/pages/learner-dashboard-page/new-learner-dashboard/progress-tab.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="completed-stories" *ngIf="completedStoriesList.length !== 0">
+  <div class="completed-stories-heading">
+    <p class="completed-stories-title" [innerHTML]="'I18N_LEARNER_DASHBOARD_STORIES_COMPLETED_SECTION' | translate">
+    </p>
+    <span class="completed-stories-length">
+      {{completedStoriesList.length}}
+    </span>
+    <img [src]="getStaticImageUrl('/learner_dashboard/star.svg')" alt="star image" class="oppia-svg-star-image">
+  </div>
+  <div class="scroll-content">
+    <div class="completed-stories-content" [ngStyle]="checkMobileView() && {'width': width + 'px'}">
+      <div *ngFor="let tile of completedStoriesList" class="story-tile">
+        <oppia-learner-story-summary-tile [storySummary]="tile">
+        </oppia-learner-story-summary-tile>
+      </div>
+    </div>
+  </div>
+</div>
+<style>
+  .completed-stories {
+    background-color: #fff;
+    border-radius: 4px;
+    box-shadow: 0 0 4px #0000001a;
+    margin: auto;
+    margin-top: 15px;
+    min-height: 342px;
+    opacity: 1;
+    padding: 0 0 30px 40px;
+    width: 736px;
+  }
+  .completed-stories-content {
```

Comment:
Ditto in this file also(you can use `.completed-stories`  as parent class).

Reply:
Done.